src/read/_s3.js

```diff
@@ -177,7 +178,7 @@ let staticAssets
 let staticManifest = join(process.cwd(), 'node_modules', '@architect', 'shared', 'static.json')
 if (staticAssets === false) { /* noop */ }
 else if (existsSync(staticManifest) && !staticAssets) {
-  staticAssets = JSON.parse(readFileSync(staticManifest))
+  staticAssets = JSON.parse(readFileSync(staticManifest).toString())
```

Comment:
This is fine but afaik JSON.parse can parse a buffer (as long as it’s a buffer of valid JSON).